libutils/regex.c

```diff
@@ -133,7 +135,14 @@ bool StringMatchFullWithPrecompiledRegex(pcre *pattern, const char *str)
     }
 }
 
-Seq *StringMatchCaptures(const char *regex, const char *str)
+// Returns a Sequence with Buffer elements.
+
+// If return_names is set, for every capture there will be the name or
+// number of the capturing group, followed by the captured data.
```

Comment:
Suggest slight clarification that even numbered indexes will be name or number, and odd numbered will be capture data. At first I thought you meant returning some kind of pair structure.